.github/workflows/docker/Dockerfile-compile-check-centos8.1

```diff
@@ -22,5 +22,5 @@ ENV GOPROXY      "https://mirrors.aliyun.com/goproxy,direct"
 ENV GO111MODULE  on
 
 # install dcap
-RUN wget https://download.01.org/intel-sgx/sgx-linux/2.12/distro/centos8.2-server/sgx_rpm_local_repo.tgz && \
-    tar -zxvf sgx_rpm_local_repo.tgz && rm sgx_rpm_local_repo.tgz && cd sgx_rpm_local_repo && rpm -ivh libsgx-headers*.rpm libsgx-dcap-quote-verify*.rpm && rm -rf sgx_rpm_local_repo
+RUN wget https://download.01.org/intel-sgx/sgx-linux/2.13/distro/centos8.2-server/sgx_rpm_local_repo.tgz && \
+    tar -zxvf sgx_rpm_local_repo.tgz && rm sgx_rpm_local_repo.tgz && cd sgx_rpm_local_repo && rpm -ivh libsgx-headers*.rpm libsgx-dcap-quote-verify*.rpm && cd ../ && rm -rf sgx_rpm_local_repo
```

Comment:
What is the purpose of this installation? For building a working development environment?

Reply:
> What is the purpose of this installation? For building a working development environment?

`rune` depends on `libsgx-dcap-quote-verify-devel` and
`libsgx-dcap-quote-verify-devel` depends on `libsgx-dcap-quote-verify` which depends on `libsgx-headers`

Reply:
So why not just running `yum install` which can resolve the dependencies instead of hard-coding the dependencies here. Once the dependencies is changed, your code has to be changed as well, right? Using `yum install -y libsgx-dcap-quote-verify-devel`, all dependencies are automatically resolved.

Reply:
> So why not just running `yum install` which can resolve the dependencies instead of hard-coding the dependencies here. Once the dependencies is changed, your code has to be changed as well, right? Using `yum install -y libsgx-dcap-quote-verify-devel`, all dependencies are automatically resolved.

In the installation tutorial of intel `sgxsdk`， the default installation mode of `ubuntu` is `apt-get` while `centos` is `rpm -ivh`

Reply:
OK. But just run `rpm -ivh libsgx-dcap-quote-verify-devel-*.rpm`.

Reply:
> OK. But just run `rpm -ivh libsgx-dcap-quote-verify-devel-*.rpm`.

Do you mean I need to install it by mandatory without ignoring the error?